topics/persistence.md

```diff
@@ -116,7 +124,7 @@ Redis it will re-play the AOF to rebuild the state.
 ### Log rewriting
 
 As you can guess, the AOF gets bigger and bigger as write operations are
-performed.  For example, if you are incrementing a counter 100 times,
+performed. For example, if you are incrementing a counter 100 times,
```

Comment:
these double spaces in the middle of the line are not rendered in the MD result.
they're not needed in the "source" but i'm not sure we need to bother removing them.

Reply:
Yes, but I think one day someone will specifically mention a PR to remove this space?

Reply:
i'll reject it if it's in my shift..  8-)
i'm usually trying to avoid changes that are unnecessary and damage the blame log.

Reply:
OK